handlers/handlers.go

```diff
@@ -106,40 +106,35 @@ func Read(rendC RenderClient) http.HandlerFunc {
 }
 
 // Edit Handler
-func Edit(rendC RenderClient) http.HandlerFunc {
+func Edit(rendC RenderClient, cfg config.Config) http.HandlerFunc {
 	return func(w http.ResponseWriter, req *http.Request) {
-		edit(w, req, rendC)
+		edit(w, req, rendC, cfg.SiteDomain)
 	}
 }
 
 // acceptAll handler for accepting all possible cookies
-func acceptAll(w http.ResponseWriter, req *http.Request) {
+func acceptAll(w http.ResponseWriter, req *http.Request, siteDomain string) {
 	ctx := req.Context()
 	cp := cookies.Policy{
 		Essential: true,
 		Usage:     true,
 	}
-	reqUrl, err := url.Parse(req.URL.Path)
-	if err != nil {
-		log.Event(ctx, "unable to parse url", log.Error(err))
-		setStatusCode(req, w, err)
-		return
-	}
-	cookies.SetPolicy(w, cp, reqUrl.Hostname())
-	cookies.SetPreferenceIsSet(w, reqUrl.Hostname())
+
+	cookies.SetPolicy(w, cp, siteDomain)
+	cookies.SetPreferenceIsSet(w, siteDomain)
 	referer := req.Header.Get("Referer")
 	if referer == "" {
 		err := errors.New("cannot redirect due to no referer header")
 		log.Event(ctx, "unable to parse url", log.Error(err))
 		setStatusCode(req, w, err)
 		return
 	}
-	log.Event(ctx,"redirecting to " + referer, log.INFO)
+	log.Event(ctx, "redirecting to "+referer, log.INFO)
```

Comment:
As this was already here, feel free to leave it, but our standards say that we should avoid formatted strings for event messages.  In this case we could add the referer to a `log.Data{}` option.

Reply:
Ah this was copied over from another repo. Have killed this line off as don't think this log is very useful